README.md

```diff
@@ -1,46 +1,84 @@
-# HomePlug Power Line Communication Exporter for Prometheus
+# HomePlug exporter for Prometheus
 
-Exports HomePlug PLC network and station statistics via HTTP for Prometheus consumption.
+Prometheus exporter for [HomePlug](https://en.wikipedia.org/wiki/HomePlug)
+Power Line Communication (PLC) devices.
 
-Help on flags:
+It collects and exports metrics about the HomePlug network and stations.
+
+Tested with TP-Link and Devolo devices but should work with any device using a
+Qualcomm Atheros chipset such as QCA6410, QCA7000, and QCA7420.
+
+Currently it does not support other chipset as it uses Atheros-specific
```

Comment:
Last nit (hopefully). I should really review PRs when I'm more awake :sleeping: 
```suggestion
Currently it does not support other chipsets as it uses Atheros-specific
```

Reply:
No worries, thanks for the attention to detail!